pkg/common-controller/snapshot_controller.go

```diff
@@ -421,6 +421,7 @@ func (ctrl *csiSnapshotCommonController) syncUnreadySnapshot(snapshot *crdv1.Vol
 	klog.V(5).Infof("getDynamicallyProvisionedContentFromStore for snapshot %s", uniqueSnapshotName)
 	contentObj, err := ctrl.getDynamicallyProvisionedContentFromStore(snapshot)
 	if err != nil {
+		klog.V(4).Infof("getDynamicallyProvisionedContentFromStore[%s]: error when get content %v", uniqueSnapshotName, err)
```

Comment:
Change msg to "error when get content for snapshot" as you gave the snapshot name here.